sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client.cpp

```diff
@@ -127,4 +125,24 @@ Response<KeyVaultCertificate> CertificateClient::GetCertificateVersion(
   return Azure::Response<KeyVaultCertificate>(std::move(value), std::move(rawResponse));
 }
 
+CreateCertificateOperation CertificateClient::StartCreateCertificate(
+    std::string const& name,
+    CertificateCreateParameters const& parameters,
+    Azure::Core::Context const& context) const
+{
+  auto payload = CertificateCreateParametersSerializer::Serialize(parameters);
+  Azure::Core::IO::MemoryBodyStream payloadStream(
+      reinterpret_cast<const uint8_t*>(payload.data()), payload.size());
+
+  auto request = CreateRequest(
+      HttpMethod::Post, {CertificatesPath, name, CertificatesCreatePath}, &payloadStream);
+
+  auto rawResponse = SendRequest(request, context);
+  auto value = KeyVaultCertificate();
+  value.Properties.Name = name;
+  auto responseT = Azure::Response<KeyVaultCertificate>(std::move(value), std::move(rawResponse));
```

Comment:
I see that you created `payloadStream` on the stack, and you return the response from the function. I don't remember how it works now, but just in case, check that response won't end up holding a dangling pointer to the `payloadStream`.